Solutions/Infoblox NIOS/Parser/Infoblox_allotherdnsTypes.txt

```diff
@@ -1,13 +1,14 @@
-// Title:           Infoblox NIOS Data Parser
+// Title:           Infoblox parser for type - DNS(other then zone, client,gss)
 // Author:          Microsoft
 // Version:         1.1
-// Last Updated:    01/03/2022
+// Last Updated:    20/05/2022
 // Comment:         Updated to only parse the MSG (RFC3164) part of the Syslog message. Excluded the Header.
 //
 // DESCRIPTION:
-// This parser takes raw Infoblox NIOS DHCP and DNS logs from a Syslog data stream and parses the data into a normalized schema
+// This parser comprised of function - Infoblox_allotherdnstypes
 //
 // USAGE:
+// USAGE:
 // 1. Open Log Analytics/Azure Sentinel Logs blade. Copy the query below and paste into the Logs query window. 
```

Comment:
The usage instructions are not needed here and confusing/incorrect (as deploying any one function would not help). True for all parsers.

Reply:
usage instruction is removed